runtime/mainnet/src/config/xcm.rs

```diff
@@ -211,3 +213,54 @@ impl cumulus_pallet_xcm::Config for Runtime {
 	type RuntimeEvent = RuntimeEvent;
 	type XcmExecutor = XcmExecutor<XcmConfig>;
 }
+
+#[cfg(test)]
+mod tests {
+	use std::any::TypeId;
+
+	use super::*;
+
+	// Only reserve accepted is the relay asset from Asset Hub.
+	#[test]
+	fn reserves() {
+		assert_eq!(
+			TypeId::of::<<XcmConfig as xcm_executor::Config>::IsReserve>(),
+			TypeId::of::<NativeAssetFrom<AssetHub>>(),
+		);
+	}
+
+	#[test]
+	fn asset_hub_as_relay_asset_reserve() {
+		assert!(TrustedReserves::contains(
+			&Asset::from((AssetId::from(Parent), Fungibility::from(100u128))),
+			&AssetHub::get(),
+		));
+	}
+
+	#[test]
+	fn relay_as_relay_asset_reserve_fails() {
+		let relay_asset = Asset::from((AssetId::from(Parent), Fungibility::from(100u128)));
+		assert!(!TrustedReserves::contains(&relay_asset, &Parent.into()));
+	}
+
+	#[test]
+	fn decline_native_assets() {
```

Comment:
Could use a comment to point that this is every other native asset but the system one.